WcaOnRails/app/views/static_pages/faq.html.erb

```diff
@@ -22,18 +22,18 @@ faq_variables = {
 
   <%
     @questions = []
-    def add_answer(question, &block)
-      @questions << question
-      render layout: "faq_item", locals: { question: question }, &block
+    def add_answer(question_data, &block)
+      @questions << question_data
+      render layout: "faq_item", locals: question_data, &block
     end
   %>
   <% @answers_markup = capture do %>
     <ul id="faq-list" class="list-group">
 
 
-      <% t('faq.answers').values.each do |faq| %>
-        <%= add_answer faq[:title] do %>
-          <% faq[:content].values.each do |paragraph| %>
+      <% t('faq.answers', locale: :en).each do |key, faq| %>
```

Comment:
Clever fix! This is kind of subtle, though. Would you be willing to add a quick comment explaining why we do it this way? We want to fall back to English if a question is not yet translated, and we want the urls to stay the same.